app/src/processing/app/Mode.java

```diff
@@ -1126,11 +1186,46 @@ public DefaultMutableTreeNode buildSketchbookTree(){
     return sbNode;
   }
   
+  
+  
   protected JFrame sketchbookFrame;
+ 
+  /**Function for rebuilding the tree in case user presses SPACE or
+  *  Sketchbook Folder is changed in Preferences. Also this helps if
+  *  a sketch is deleted from this folder not by means of Processing application window
+  *  but externally. The user can press SPACE to see changes
+  **/
+  public void rebuildTree()
+  {   
```

Comment:
Please follow the layout conventions used elsewhere in the code (opening brace on previous line).